ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -23,6 +23,10 @@ export default class TestDataModal extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      // "Pinned" values to be used in dropdown menus
```

Comment:
I think that instead of `pinned` object we can use a `pinnedProjects` Array.
The code would be more direct this way. No need for `pinned.projects`.